renderer/constants/Constants.js

```diff
@@ -1,50 +0,0 @@
-import keyMirror from 'keymirror';
-
-export const VERSION = '2.1.0';
-export const PeripheralTypes = keyMirror({
-  MOTOR_SCALAR: null,
-  SENSOR_BOOLEAN: null,
-  SENSOR_SCALAR: null,
-  LimitSwitch: null,
-  LineFollower: null,
-  Potentiometer: null,
-  Encoder: null,
-  ColorSensor: null,
-  MetalDetector: null,
-  ExampleDevice: null,
-  ServoControl: null,
-  YogiBear: null,
-  RFID: null,
-  BatteryBuzzer: null,
-  TeamFlag: null,
-  GameValues: null,
-  master: null,
-  PolarBear: null,
-  KoalaBear: null,
-});
-
-export const ActionTypes = keyMirror({
-  FIELD_CONTROL: null,
-  UPDATE_BATTERY: null,
-  UPDATE_GAMEPADS: null,
-  UPDATE_EDITOR: null,
-  UPDATE_FILENAMES: null,
-  UPDATE_FILENAME: null,
-  UPDATE_PERIPHERAL: null,
-  UPDATE_STATUS: null,
-  UPDATE_CONNECTION: null,
-  UPDATE_FC_CONFIG: null,
-  SAVE_FILE: null,
-  OPEN_FILE: null,
-  CLEAR_EDITOR: null,
-  UPDATE_CONSOLE: null,
-  UPDATE_TIMER: null,
-  UPDATE_HEART: null,
-  UPDATE_ROBOT: null,
-  UPDATE_MATCH: null,
-  UPDATE_MASTER: null,
-  ADD_ALERT: null,
-  REMOVE_ALERT: null,
-  CLEAR_CONSOLE: null,
-  runtime_version: null,
-});
```

Comment:
Is there a reason why you deleted this and didn't add to consts/Constants.js? This part is needed for `FieldActions.ts`. Should I copy this back to the document?

Reply:
Ahh yes, for this instead of `ActionTypes.UPDATE_MASTER` like it is now, you can use the new pattern we're using.

Reply:
You're saying that we are putting like `Field_Control` and `Update_FC_Config` inside enum in `consts/actions.ts` right? Just wanted to make sure.

Reply:
Yes, this way it is consistent with the pattern we did with the other file already.